.flowconfig

```diff
@@ -0,0 +1,7 @@
+[ignore]
+
+[include]
+
+[libs]
+
+[options]
```

Comment:
[compare to our React Native one](https://github.com/artsy/emission/blob/master/.flowconfig) - wonder if that can be simplified now.